README.rst

```diff
@@ -37,7 +39,26 @@ Create a build directory, then, from it, run cmake and make::
     cmake ..
     make
 
-Linux users can install Dwarf Therapist by running ``make install``. The destination directory can be changed by passing the ``-DCMAKE_INSTALL_PREFIX=/path/to/dwarftherapist`` option to cmake (e.g. ``cmake -DCMAKE_INSTALL_PREFIX=~/.local ..`` for a user-local installation).
+For Linux Users:
+
+You can install Dwarf Therapist by running ``make install``. The destination directory can be changed by passing the ``-DCMAKE_INSTALL_PREFIX=/path/to/dwarftherapist`` option to cmake (e.g. ``cmake -DCMAKE_INSTALL_PREFIX=~/.local ..`` for a user-local installation).
+
+For Mac users:
+
+Ensure prerequsites are installed::
+
+    brew install qt@5.5
+    brew install cmake
+    brew cask install basictex
```

Comment:
If you want to build the manual, you need to add -DBUILD_MANUAL=ON in cmake parameters. Is basictex enough? On fedora I need: texlive-bibtex, texlive-cm, texlive-latex-bin, texlive-makeindex, texlive-mfware, texlive-preprint, texlive-sidecap, texlive-wrapfig.